Python/initconfig.c

```diff
@@ -3026,3 +3022,32 @@ _Py_DumpPathConfig(PyThreadState *tstate)
 
     _PyErr_Restore(tstate, exc_type, exc_value, exc_tb);
 }
+
+
+/* --- _PyInterpreterConfig ---------------------------------- */
+
+void
+_PyInterpreterConfig_Clear(_PyInterpreterConfig *config)
+{
+    config->allow_fork = 0;
+    config->allow_subprocess = 0;
+    config->allow_threading = 0;
+}
+
+
+PyStatus
+_PyInterpreterConfig_Copy(_PyInterpreterConfig *config,
+                          const _PyInterpreterConfig *config2)
+{
+    _PyInterpreterConfig_Clear(config);
+
+#define COPY_ATTR(ATTR) config->ATTR = config2->ATTR
+
+    COPY_ATTR(allow_fork);
+    COPY_ATTR(allow_subprocess);
+    COPY_ATTR(allow_threading);
+
+#undef COPY_ATTR
+
```

Comment:
Could a simple assignment be used instead?

```suggestion
    *config = *config2;
```